sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -139,6 +141,8 @@ private[sql] class SQLConf extends Serializable {
    */
   private[spark] def codegenEnabled: Boolean = getConf(CODEGEN_ENABLED, "false").toBoolean
 
+  private[spark] def useSqlSerializer2: Boolean = getConf(USE_SQL_SERIALIZER2, "false").toBoolean
```

Comment:
Also do we want to turn it on by default?  Its easy to turn off if we find bugs.